packages/sync-actions/src/product-actions.js

```diff
@@ -175,7 +174,7 @@ function _buildSetAttributeAction(
   attribute,
   sameForAllAttributeNames
 ) {
-  if (!attribute) return undefined
+  if (!attribute || !diffedValue) return undefined
```

Comment:
since this package already super confusing, please feel free to add some comment or ask AI to add it :)

Reply:
here 😄 -> https://github.com/commercetools/nodejs/pull/1975/commits/9a4c03654591add687d44bde3ff1792dff463273